package.json

```diff
@@ -63,7 +63,8 @@
     "passport-local": "^1.0.0",
     "password-generator": "^2.0.2",
     "sequelize": "^3.24.3",
-    "socket.io": "^1.4.6"
+    "socket.io": "^1.4.6",
+    "socket.io-client": "^1.5.0"
```

Comment:
The `socket.io-client` package should be in the webapp's package.json, not at all in the API server one !